docs/connector-development/tutorials/build-a-connector-the-hard-way.md

```diff
@@ -975,6 +997,7 @@ tests:
 Then from the connector module directory run
 
 ```bash
+chmod 755 acceptance-test-docker.sh
```

Comment:
@sherifnada since this one is a shell script, the permissions given during the `chowning generated files` weren't enough for it to run for me. This is how I got around it to continue (copied permissions from `generate.sh` which ran file)

I don't think this is the right way to do this, hence the draft pr, would prefer just changing the file itself that gets copied or something in the generate script to give it the right permissions. Wdyt?

Reply:
+1 from me on chmoding the file in git

Reply:
Cool, always a thing I want to double check about because... permissions. But if we have those permissions for the generate script, figured it should probably be fine. Have to make sure chowning stuff doesn't mess with those git permissions though.